vra/resource_image_profile_test.go

```diff
@@ -0,0 +1,127 @@
+package vra
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/vmware/vra-sdk-go/pkg/client/image_profile"
+)
+
+func TestAccVRAImageProfileBasic(t *testing.T) {
+	rInt := acctest.RandInt()
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheckMachine(t) },
```

Comment:
I think **testAccPreCheckMachine()** requires flavor, and project besides image. The test uses only the cloud account name and image. Please create a separate zoom with only those that are required for creating an Image Profile.

Reply:
done!